package.json

```diff
@@ -5,16 +5,18 @@
   "main": "ping-centre.js",
   "license": "MPL-2.0",
   "scripts": {
-    "test": "npm-run-all test:*",
-    "test:code": "make test",
-    "test:lint": "eslint ."
+    "bundle": "webpack",
+    "test": "npm run bundle && npm run test:node && npm run test:browser && npm run test:lint",
+    "test:node": "istanbul cover _mocha -- --recursive",
+    "test:browser": "karma start test/karma.conf.js",
+    "travis": "npm test && cat ./coverage/lcov.info | coveralls",
+    "test:lint": "eslint . --ext .js"
```

Comment:
Because of the order of build+lint operations,  you'll need to add an .eslintignore file with at least the following paths (since it looks like Travis-CI is linting generated files):

```
coverage
dist
```